.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2026-01-23 - Prevent SQL Injection with f-strings in duckdb/sqlite
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)